### ISSUE TITLE
As an admin, I want a way to disable a restaurant + the restaurant page is still accessible (no 404)

### ISSUE THREAD

**Issue by Atala** [coopcycle/coopcycle-web#4427]
![image](https://github.com/coopcycle/coopcycle-web/assets/4426585/bbb7ff1b-069c-4b16-bbc7-9a55017a7f38)

- The existing "deactivated" switch was meant to be use before going to production or after stopping working with the restaurant. Thus when you turn it off the restaurant page is not accessible anymore ( 404 ). this will break for example a link from instagram. 

- What Naofood wants is a way to disable ordering on the restaurant but the page still shows.

- The workaround they use now is to disable both delivery and click collect in the fulffilment methods.

**Comment by vladimir-8** [comment]
As I understand https://demo.coopcycle.org/admin/restaurants/10/planning is meant exactly for this use case

**Comment by Paul-Eraman-CoopCycle** [comment]
Yes, but its true that the buton to do this essily is in the apo, not in the dispatcher web part

**Comment by Atala** [comment]
![image](https://github.com/coopcycle/coopcycle-web/assets/4426585/4e28b081-dada-46e4-b18d-1b1679555e6a)

it is here (but you may know that already)

yes i guess the feature is right there, we just miss a button to easily close the next day in the web part? (i have no idea how it looks in the app). also maybe the activate/deactivate should not be accessible to the restaurant owner?

**Comment by Paul-Eraman-CoopCycle** [comment]
Yes but that buton is buried behind tons of clicks, needs putting in dates...

Heres the app buton![image](https://github.com/coopcycle/coopcycle-web/assets/77277854/b8947814-ad3c-4158-9f73-a62e7d44b7c1)

**Comment by Paul-Eraman-CoopCycle** [comment]
My take:

Basically this is a "aaaa fuck" minute to minute dispatch task and therefore should be in the dispatch panel somewhere and easily accessible. Orders dashboard also would be fine 

Whereas that calendar is for preprograming stuff for next week (week-monthly actions not emergencies)

**Comment by Atala** [comment]
yes sure sure it was meant for holydays when it was built

**Comment by Paul-Eraman-CoopCycle** [comment]
Also it should be for the shop too
In the app its a great feature for the restaurant

**Comment by Paul-Eraman-CoopCycle** [comment]
Could we just add this same button next to the "disable" button? seems easy but maybe im wrong

**Comment by Atala** [comment]
yes indeed

there should be :
 - a button that disable a restaurant (restaurant not ready, restaurant left the platform)
 - a button to temporarily the restaurant (disable the "add to cart button" for example or something like this)

this should be two different on/off fields or CLOSED/DISABLED/OPEN state in the database

**Comment by vladimir-8** [comment]
> this should be two different on/off fields or CLOSED/DISABLED/OPEN state in the database

Let's maybe not touch the database and just do what the app does? (Add a button that will make a closing rule for today: https://github.com/coopcycle/coopcycle-web/blob/836f656061b5c7811cb4aa714e30328522d6e265/src/Action/Restaurant/Close.php#L8)

**Comment by Paul-Eraman-CoopCycle** [comment]
Thats what i was thinking, that would be very simple right?

**Comment by vladimir-8** [comment]
I'd say one day
